crates/nu-command/src/conversions/into/filesize.rs

```diff
@@ -110,28 +110,36 @@ fn into_filesize(
     )
 }
 
-pub fn action(input: &Value, span: Span) -> Value {
+pub fn action(input: &Value, _span: Span) -> Value {
+    let value_span = input.span().expect("unable to get input value span");
```

Comment:
this will cause nu to crash if anyone passes an error value in. I think instead, you want to just call `if let Some(span) = input.span() { ... }`

Reply:
ok. how's it look now?

Reply:
definitely closer. I think one last tweak and you've got it